index.html

```diff
@@ -8,88 +8,89 @@
+          <strong class="project__category">branding & web</strong>
+          <section class="project__container">
+            <h3 class="project__name">pineapp.</h3>
+            <p class="project__description">
               <span>Period: March-April 2016</span>
               <span>Team: John (UI), Edwin (UX), Jessica (Front-end)</span>
             </p>
-            <a href="/">visit the website</a>
+            <a href="/" class="project__link">visit the website</a>
           </section>
         </article>
         <article class="project">
-          <img src="assets/healthydining.jpg" alt="broccoli image">
-          <strong>branding & mobile</strong>
-          <section class="project-description">
-            <h3>healty dining.</h3>
-            <p>
+          <img src="assets/healthydining.jpg" class="project__photo" alt="broccoli image">
+          <strong class="project__category">branding & mobile</strong>
+          <section class="project__container">
+            <h3 class="project__name">healty dining.</h3>
+            <p class="project__description">
               <span>Period: March-April 2016</span>
               <span>Team: John (UI), Edwin (UX), Jessica (Front-end)</span>
             </p>
-            <a href="/">visit the website</a>
+            <a href="/" class="project__link">visit the website</a>
           </section>
         </article>
         <article class="project">
-          <img src="assets/smartcity.jpg" alt="building image">
+          <img src="assets/smartcity.jpg" class="project__photo" alt="building image">
           <strong>branding & web</strong>
-          <section class="project-description">
-            <h3>smart city.</h3>
-            <p>
+          <section class="project__container">
+            <h3 class="project__name">smart city.</h3>
+            <p class="project__description">
               <span>Period: March-April 2016</span>
               <span>Team: John (UI), Edwin (UX), Jessica (Front-end)</span>
             </p>
-            <a href="/">visit the website</a>
+            <a href="/" class="project__link">visit the website</a>
           </section>
         </article>
     </section>
-    <section class="team">
-      <h2>team</h2>
-      <figure class="team-image">
-        <img src="assets/team.jpg" alt="team members photo">
-        <figcaption>
-         <span>Meet our creative team:</span>
-         <span>Markus, Elisabeth, George & Micheal.</span>
+    <section class="main-section__team">
+      <h2 class="team__heading">team</h2>
+      <figure class="team__photo-container">
+        <img src="assets/team.jpg" class="team__photo" alt="team members photo">
+        <figcaption class="team__people">
+          <span>Meet our creative team:</span>
+          <span>Markus, Elisabeth, George & Micheal.</span>
         </figcaption>
       </figure>
       <div class="team-description">
```

Comment:
To jest component `team-description` czy miał być element `team__description`? :)

Reply:
Myślałam jako o komponencie, a Ty? :) wtedy później powinno być "team-description__title" pewnie.